conjure-java-client-verifier/src/test/resources/ignored-test-cases.yml

```diff
@@ -43,6 +43,7 @@ client:
    - 'null'
    receiveListAnyAliasExample:
    - 'null'
+   - '[null, 0, "content", true, [1,2,3], {"key":3}]' # null is not a valid any value, see conjure-verification#141
```

Comment:
Fix: https://github.com/palantir/conjure-verification/pull/141

Reply:
Surprised this was working before.. was it just deserializing the `null` to `Optional.empty()`?

Reply:
Nope, Alias<X> isn't any different from deserializing a raw X, so it used an `ArrayList<Object>` which does not do any validation.

Reply:
Argh.. I really dislike this magic inversion of control inherent in reflection-based serde.